.github/workflows/ci.yml

```diff
@@ -44,11 +44,33 @@ jobs:
           push: true
           tags: ${{ env.TEST_TAG }}
 
+      - name: Pull tests
```

Comment:
You need to checkout in a subfolder ? Here you overwrite the first checkout repo ?

Reply:
I don't think that's the problem, currently I have these commands to be able to know what command is the bash script executing
```bash
echo "INPUT_LANG: $INPUT_LANG"
echo "sw $INPUT_PATH -e $EXTENSIONS"
```
and when I try to run the docker localy I get
```
INPUT_LANG: rs js
sw test/test -e rs -e js
```
but when it runs in the github action I get
```
INPUT_LANG: $INPUT_LANG
sw $INPUT_PATH -e $INPUT_LANG
```

Reply:
it's not a problem as the container is built before overwriting the first checkout repo and building the docker is the only thing we need to do from the first checkout